src/templates/doe_workflow_central_ducted_split_heat_pump.mdx

```diff
@@ -0,0 +1,439 @@
+        <NumberInput label="Dual Fuel Switch Over Temperature (&deg;F)" min={0} path="dual_fuel_switch_over_temperature" /> 
+
+        <PhotoInput id="mfg_extended_cooling_performance_table_photo" label="MFG's cooling performance table – Photo" allowUpload>
+            Upload a photo of the MFG's extended cooling performance table showing sensible, latent, or SHR
+        </PhotoInput>
+        
+        ## Pre-installation Additional Comments
+        <StringInput label="Additional Comments" path="comments_ductwork_concluding_summary"  hint="E.g., Return air filter grille upsizing" />
+    </Tab>
+    <Tab eventKey="installation" title="Installation">
+        <DateInput label="Installation Date" path="install_date" />
+        ## Equipment
+        <PhotoInput id="odu_nameplate_photo" label="ODU nameplate – Photo" >
+            Take a photo of the Outdoor Unit (ODU) nameplate.
+        </PhotoInput>
+        <PhotoInput id="idu_nameplate_photo" label="IDU nameplate – Photo">
+            Take a photo of the Indoor Unit (IDU) nameplate.
+        </PhotoInput>
+        ## Electrical
+        <PhotoInput id="odu_circuit_breaker_photo" label="ODU circuit breaker – Photo">
+            Take a photo showing the amperage of the breaker.
+        </PhotoInput>
+        <PhotoInput id="idu_circuit_breaker_photo" label="IDU circuit breaker – Photo">
+            Take a photo showing the amperage of the breaker.
+        </PhotoInput>
+        
+        <StringInput label="Circuit Breaker Notes" path="comment_circuit_breaker_notes" hint="E.g., wire size modifications, or shared or repurposed circuits." />
+        ## Installation Tests
+        <PhotoInput id="nitrogen_pressure_test_setup_photo" label="Nitrogen Pressure Test Setup – Photo">
+            Take a photo of the nitrogen pressure test setup. Test to 500 PSI or manufacturer's recommended test pressure.
+        </PhotoInput>
+        <PhotoInput id="nitrogen_pressure_test_result_photo" label="Nitrogen Pressure Decay Test Results – Photo">
+            Take a photo of the pressure gauge showing the temperature-compensated pressure decay after 10 minutes. Please use a digital pressure gauge.
+        </PhotoInput>
+        <PhotoInput id="vaccum_decay_test_setup_photo" label="Vacuum Decay Test Setup – Photo">
+            Take a photo of the vacuum/evacuation setup. Use core removal tool and minimize number of fittings.
+        </PhotoInput>
+        <PhotoInput id="vaccum_decay_test_results_photo" label="Vacuum Decay Test Results – Photo">
+            Take a photo of the vacuum decay test results. Use a digital micron gauge. The system must be isolated from the vaccum pump. The vaccum must not rise above 500 microns in 15 minutes.
+        </PhotoInput>
+        ## Refrigerant Adjustments
+        <PhotoInput id="refrigerant_adjustments_if_applicable_photo" label="Refrigerant Adjustments (if applicable) – Photo">
```

Comment:
I don't think you need to include "if_applicable" in this `id`.
Check for other instances of this.

Reply:
removed 'if_applicable' for this component and no other similar occurrences in both the templates.